modules/disk-uploader/pkg/certificate/certificate_test.go

```diff
@@ -0,0 +1,120 @@
+package certificate_test
+
+import (
+	"context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1beta1 "kubevirt.io/api/export/v1beta1"
+	kubevirtfake "kubevirt.io/client-go/generated/kubevirt/clientset/versioned/fake"
+	"kubevirt.io/client-go/kubecli"
+
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/kubevirt/kubevirt-tekton-tasks/modules/disk-uploader/pkg/certificate"
+)
+
+var _ = Describe("Certificate", func() {
+	var (
+		ctrl           *gomock.Controller
+		vmExportClient *kubevirtfake.Clientset
+		virtClient     kubecli.KubevirtClient
```

Comment:
Is this client used somewhere?

Reply:
Yes, `virtClient` is passed as a parameter to the certificate's functions.

Reply:
And also `vmExportClient` is used to create VME.

Reply:
I see